### DISCUSSION TITLE
[Feature Request] Sleep when chapter is finished

### DISCUSSION THREAD

**Discussion by TechnicJelle** [PaulWoitaschek/Voice#1323] [Ideas]
A useful option would be to add a button in the sleep timer to play until the end of the chapter (or file).

**Comment by daquinoaldo** [comment]
@PaulWoitaschek any news about this feature? I would love it!

**Comment by Shuunen** [comment]
Would be great with this feature, I'll be glad to help but I'm not a native mobile app developer

**Comment by snkfxx** [comment]
Please consider adding this feature. IMO the only thing missing in this great app.

**Comment by Sammeeey** [comment]
I am also interested in this feature but not able to implement it.

**Comment by eugenkaltenegger** [comment]
Would also love to see this feature - for me that's the one main thing missing in this otherwise great app.

**Comment by theAkito** [comment]
Great feature, but the maintainer of this project seems currently inactive. 🙁

**Comment by srphillips** [comment]
I would love to see this get implemented!  There's currently a pull request open that the maintainer is at least speaking with the submitter on for it here: https://github.com/PaulWoitaschek/Voice/pull/1647

**Comment by yanneves** [comment]
Was looking for this to and it's in development in a new pull request in https://github.com/PaulWoitaschek/Voice/pull/2838
